packages/frontend/src/components/common/Filters/FilterInputs/AutoComplete/MultiAutoComplete.tsx

```diff
@@ -77,21 +86,41 @@ const MultiAutoComplete: FC<Props> = ({
             ),
         [isSearching],
     );
-    const onItemSelect = useCallback(
+
+    const handleItemSelect = useCallback(
         (value: string) => {
-            onChange(toggleValueFromArray(values, value));
+            onChange(toggleValueFromArray(values, value, normalize));
         },
         [onChange, values],
     );
-    const onRemove = useCallback(
+
+    const handleRemove = useCallback(
         (selectedValue: React.ReactNode) => {
             onChange(values.filter((v: string) => v !== selectedValue));
         },
         [onChange, values],
     );
+
+    const handleBlur = useCallback(
+        (value?: string) => {
+            if (!value || value === '') return;
+
+            setSearch('');
+            if (!values.map(normalize).includes(normalize(value))) {
+                const existingOptionMatch = [...options].find((option) =>
```

Comment:
So this guesses what the user wanted to select from the options list ? eg: user types `jose` and on blur it would select `Jose`

Reply:
@ZeRego correct, I chose that behaviour because if you type in jose it matches with Jose from the dropdown list and clicking enter also picks up Jose as a final value.